src/odemis/driver/actuator.py

```diff
@@ -1721,7 +1721,8 @@ class RotationActuator(model.Actuator):
     0 -> cycle. It also supports to pass an offset to the position conversion, via the MD_POS_COR metadata.
     """
 
-    def __init__(self, name, role, children, axis_name, cycle=2*math.pi, inverted=None, **kwargs):
+    def __init__(self, name, role, children, axis_name, cycle=2*math.pi,
+                 ref_start=2*math.pi*0.05, inverted=None, **kwargs):
```

Comment:
I'd suggest to default to `None`, and if it's `None`, replace by `self._cycle*0.05`. That makes it more likely to default to a good value.